docs/src/userguide/parallelism.rst

```diff
@@ -187,7 +187,7 @@ enable OpenMP.  For gcc this can be done as follows in a ``setup.py``:
 
         .. literalinclude:: ../../examples/userguide/parallelism/setup_pyx.py
 
-For Microsoft Visual C++ compiler, use ``'/openmp'`` instead of ``'-fopenmp'``.
+For the Microsoft Visual C++ compiler, the correct flag is ``'/openmp'`` instead of ``'-fopenmp'`` for the ``'extra_compile_args'`` option, and you should not pass either of the arguments ``'/openmp'`` or ``'-fopenmp'`` to the ``'extra_link_args'`` option to prevent an unnecessary warning.
```

Comment:
```suggestion
For the Microsoft Visual C++ compiler, use ``'/openmp'`` instead of ``'-fopenmp'`` for the ``'extra_compile_args'`` option. There is no need to add any flags to the ``'extra_link_args'`` option.
```

I've just tried to make it a little more concise here.

Thanks a lot for the advice - it's very useful to know!

Reply:
Thank you for the clarification. To be even more clear I would suggest to still add that there is no need for the specific flag "/openmp" in extra_link_args, however, there could still be a need for other flags.

Reply:
I've gone back and edited my suggestion to "no need to add an OpenMP flag"

Reply:
Thanks! By mistake you wrote 'flags' and not 'flag'.